client/components/tooltip/style.scss

```diff
@@ -1,3 +1,7 @@
+@import "@automattic/typography/styles/variables";
```

Comment:
Where is this import being used within this file? If we're not using this, then let's omit this change from the PR.

Reply:
Well observed! Thanks!